patches/chrome-android-java-src-org-chromium-chrome-browser-customtabs-features-toolbar-CustomTabToolbar.java.patch

```diff
@@ -1,13 +1,13 @@
 diff --git a/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/toolbar/CustomTabToolbar.java b/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/toolbar/CustomTabToolbar.java
-index 1be17e15f4f6c3c8b7f4a0a3ee37161f35c16dcc..47087c7946b7a0581f1d6c516e18f52fbc9a50b1 100644
+index 1be17e15f4f6c3c8b7f4a0a3ee37161f35c16dcc..34f9551f8de23d0fe716b8e1af13811a2443dd60 100644
 --- a/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/toolbar/CustomTabToolbar.java
 +++ b/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/toolbar/CustomTabToolbar.java
 @@ -97,7 +97,7 @@ import java.util.regex.Pattern;
  /**
   * The Toolbar layout to be used for a custom tab. This is used for both phone and tablet UIs.
   */
 -public class CustomTabToolbar extends ToolbarLayout implements View.OnLongClickListener {
-+public class CustomTabToolbar extends BraveToolbarLayout implements View.OnLongClickListener {
++public class CustomTabToolbar extends org.chromium.chrome.browser.toolbar.top.BraveToolbarLayout implements View.OnLongClickListener {
```

Comment:
just like c++ patches, we should convert these to asm if we need to manually change them

Reply:
Addressed here https://github.com/brave/brave-core/commit/c3c611b45d6b7266a100913648a1c5530890ecc0